src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationServiceRunner.vb

```diff
@@ -53,6 +54,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
             Return ImmutableArray(Of ISymbol).Empty
         End Function
 
+        Protected Overrides Function IsInvocationExpression(syntaxNode As SyntaxNode) As Boolean
+
+            Return syntaxNode.IsKind(SyntaxKind.InvocationExpression)
+        End Function
+
+        Protected Overrides Function IsLambdaExpression(syntaxNode As SyntaxNode) As Boolean
+
```

Comment:
remove

Reply:
I'm not a fan of those empty lines. I just found them in the original code. Was it an obsolete code style?

Reply:
yeah... that's super bizarre... i'm not sure how that happened.  def can remove :)  (and can get rid of all my comments).

Reply:
OK. Thanks! Will remove all the empty lines